core/src/main/assembly/bundle.xml

```diff
@@ -0,0 +1,19 @@
+<assembly xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+ xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.1 http://maven.apache.org/xsd/assembly-1.1.1.xsd">
+ <id>bundle</id>
+ <formats>
+  <format>jar</format>
+ </formats>
+ <includeBaseDirectory>false</includeBaseDirectory>
+ <fileSets>
+  <fileSet>
+   <directory>${project.build.directory}/resources</directory>
+   <outputDirectory>/</outputDirectory>
+  </fileSet>
+ </fileSets>
+ <dependencySets>
+  <dependencySet>
+   <unpack>true</unpack>
```

Comment:
Not sure about 
<exclude>io.vertx:*</exclude>
That would exclude also vertx-service-factory
Maybe 
<exclude>io.vertx:vertx-core</exclude>
is best